README.md

```diff
@@ -1,3 +1,46 @@
 [![CI](https://github.com/embedded-dev-research/itlab_2023/actions/workflows/ci.yml/badge.svg)](https://github.com/embedded-dev-research/itlab_2023/actions/workflows/ci.yml)
 
 # itlab_2023
+# Name
+## Short description ##
+# **Building a Project**
+To build and run this project locally, follow these steps:
+
+1. **Clone the repository:**  
+   Clone this repository to your local machine using the following command:
+   ```bash
+   git clone https://github.com/embedded-dev-research/itlab_2023.git
+2. **Update submodules:**
+   Navigate to the project directory and update the submodules:
+   ```bash
+   git submodule update --init --recursive
+3. **Configure the project:**
+   Create a separate directory for configure the project and compile it:
+   ```bash
+   mkdir build
+   cd build
+   cmake ..
+    ```
+    *Note: Make sure you have CMake installed to build the project.*
+4. **Build the project:**\
+   Next, to build the project, we will need to enter the command
+    ```bash
+   cmake --build . --config Debug
+    ```
+    If you want to build in a release, change the debug to release\
+5. **Run the project**
+   After building the project, you can find the executable file in the following path from the *build* folder
+   ```bash
+   cd bin\Debug
+    ```
+   and run the file
+    ```bash
+   Reader.exe
+    ```
```

Comment:
Note that this is relevant only for Windows. Try these steps on Linux and compare